server/src/main/java/org/opensearch/action/admin/cluster/state/ClusterStateResponse.java

```diff
@@ -108,20 +108,20 @@ public boolean equals(Object o) {
         if (o == null || getClass() != o.getClass()) return false;
         ClusterStateResponse response = (ClusterStateResponse) o;
         return waitForTimedOut == response.waitForTimedOut && Objects.equals(clusterName, response.clusterName) &&
-        // Best effort. Only compare cluster state version and master node id,
+        // Best effort. Only compare cluster state version and cluster-manager node id,
         // because cluster state doesn't implement equals()
             Objects.equals(getVersion(clusterState), getVersion(response.clusterState))
-            && Objects.equals(getMasterNodeId(clusterState), getMasterNodeId(response.clusterState));
+            && Objects.equals(getClusterManagerNodeId(clusterState), getClusterManagerNodeId(response.clusterState));
     }
 
     @Override
     public int hashCode() {
-        // Best effort. Only use cluster state version and master node id,
+        // Best effort. Only use cluster state version and cluster-manager node id,
         // because cluster state doesn't implement hashcode()
-        return Objects.hash(clusterName, getVersion(clusterState), getMasterNodeId(clusterState), waitForTimedOut);
+        return Objects.hash(clusterName, getVersion(clusterState), getClusterManagerNodeId(clusterState), waitForTimedOut);
     }
 
-    private static String getMasterNodeId(ClusterState clusterState) {
+    private static String getClusterManagerNodeId(ClusterState clusterState) {
         if (clusterState == null) {
             return null;
         }
```

Comment:
Should the `getMasterNodeId` below in line 130 need to be changed to `getClusterManagerNodeId`?

Reply:
Hi @xuezhou25 , thanks a lot for your time in reviewing my PR! 👍👍.
the method `getMasterNodeId()` in line 130 is published to Maven as Java API. Please see my comment above: https://github.com/opensearch-project/OpenSearch/pull/2519#issuecomment-1074706003.
Although the normal way to rename the Java API is to deprecate old method and create new alternative method, there are too many public methods contain name "master", and it will introduce too many extra codes to deprecate them all. So I plan to rename them all in a future major version, and give the other plugins and clients enough notice before renaming.